service.go

```diff
@@ -120,6 +120,33 @@ func (c *Client) ListServices(o ListServiceOptions) (*ListServiceResponse, error
 	return &result, c.decodeJSON(resp, &result)
 }
 
+// ListServices lists existing services processing paginated responses
+func (c *Client) ListServicesPaginated(o ListServiceOptions) ([]Service, error) {
```

Comment:
Could you make the signature `ListServicesPaginated(ctx context.Context, o ListServiceOptions)` and pass the `ctx` into `pagedGet()`?